src/app/test-record/visit/visit.component.scss

```diff
@@ -0,0 +1,7 @@
+.govuk-summary-list {
+  & .govuk-summary-list__row:last-child {
+    & .govuk-summary-list__value, .govuk-summary-list__key {
+      border-bottom: none !important;
```

Comment:
same for scss, this override everything, if is like that from the design system we have been given we can have a discussion with the designers and dev but this should not be changed nor be a bug